src/MaterialDesignThemes.Wpf/HintAssist.cs

```diff
@@ -119,6 +119,17 @@ public static void SetBackground(DependencyObject element, Brush value)
         => element.SetValue(BackgroundProperty, value);
     #endregion
 
+    #region AttachedProperty: HintPaddingBrush
+    public static readonly DependencyProperty HintPaddingBrushProperty =
+        DependencyProperty.RegisterAttached("HintPaddingBrush", typeof(Brush), typeof(HintAssist), new PropertyMetadata(null));
+
+    public static Brush GetHintPaddingBrush(DependencyObject obj)
+        => (Brush)obj.GetValue(HintPaddingBrushProperty);
+
+    public static void SetHintPaddingBrush(DependencyObject obj, Brush value)
+        => obj.SetValue(HintPaddingBrushProperty, value);
```

Comment:
```suggestion
    public static Brush? GetHintPaddingBrush(DependencyObject obj)
        => (Brush)obj.GetValue(HintPaddingBrushProperty);

    public static void SetHintPaddingBrush(DependencyObject obj, Brush? value)
        => obj.SetValue(HintPaddingBrushProperty, value);
```

Reply:
Doh! I always forget this when using the built-in macros. Thanks 👍